test/fixtures/comprehensions/await/actual.js

```diff
@@ -1,4 +1,4 @@
 f() -/>
-  <- [for x of arr:
+  <- [for now x of arr:
```

Comment:
My procedure for updating pre-existing test fixtures was to always obtain the same output, which is done by adding `now`. Adding `const` would have changed the expected output. I wanted to be sure we were testing the same code paths we were testing before.

Am happy to add separate fixtures for `const` but I think we are very well covered on the different scenarios, unless you can point out a specific untested scenario?